.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Could you remove this file from your PR. We have a PR open to have VSCode ignore this.